Tests.Integration.Presentation.Web/Authorization/GetOrganizationsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Authorization
+{
+    public class GetOrganizationsTests : WithAutoFixture
+    {
+        [Theory]
+        [InlineData("Name")]
+        [InlineData("")]
+        [InlineData(null)]
+        public async Task Get_Organizations_Returns_List(string orderBy)
+        {
+            var response = await OrganizationHelper.GetOrganizationsResponseAsync(orderBy);
```

Comment:
`Get_Organizations_Returns_List`
But the test only asserts the response code :-)